### ISSUE TITLE
Can I stop Sublime highlighting my code this way:

### ISSUE THREAD

**Issue by gbritton1** [DamnWidget/anaconda#518]
![2016-08-29_18-57-07](https://cloud.githubusercontent.com/assets/12263288/18070239/92738806-6e1a-11e6-8fed-5f3540f4871a.png)

How can I stop Sublime from boxing every line like this?

**Comment by DamnWidget** [comment]
You get those boxes cos your code has linting errors, why don't you try
just to fix what the status bar tells you about the line when you have
selected it?

If you still want to disable it just set the option "anaconda_linting" as
`false` in the anaconda configuration file but that will not change the
fact that the code is erroneous or not PEP8 compliant, it just ignored the
fact...

On Aug 29, 2016 11:59 PM, "gbritton1" <notifications@github.com> wrote:

[image: 2016-08-29_18-57-07]
<https://cloud.githubusercontent.com/assets/12263288/18070239/92738806-6e1a-11e6-8fed-5f3540f4871a.png>

How can I stop Sublime from boxing every line like this?

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/DamnWidget/anaconda/issues/518>, or mute the thread

**Comment by gbritton1** [comment]
Thanks!  Didn't realize those were linting errors.  Turns out it was tab indented and I need space indented.  That led me on another journey to find out how to do that.

But it doesn't really work the way I want.  I set up a Python.sublime_settings like this:

{
	"translate_tabs_to_spaces": true,
	"tab_size": 4
}

then (do I have to restart for every change? What a drag!) reopened the file with problems.  I used search/replace to change single tabs to 4 spaces.  Fine.

I noticed in the bottom right of the status bar though that tab size was set to 4.  four what?

I thought it would change to spaces: 4.  Once I changed it in the status bar popup to turn on indentation using spaces, I got the behavior I need.

Is there a setting so that I don't have to change that to spaces: 4 by hand every time?
